Make.defs

```diff
@@ -17,13 +17,15 @@ DBGFLAG = #-ggdb
 ifndef CFLAGS
 CFLAGS = -O2
 endif
+CFLAGS := -fstack-protector-strong -D_FORTIFY_SOURCE=2 $(CFLAGS)
 CFLAGS += $(DBGFLAG) -I../include
 CFLAGS += -Wall -Wextra
 ifneq ($(WERROR),)
 CFLAGS += -Werror
 endif
 #CFLAGS += -DFAIL_RECORD
 LDFLAGS += $(DBGFLAG) -L../libs
+LDFLAGS := -Wl,-z,defs -Wl,-z,relro -Wl,-z,now $(LDFLAGS)
```

Comment:
I'd prefer to define separate variables, e.g. `HARDENING_CFLAGS` and `HARDENING_LDFLAGS`, and to include them into `CFLAGS` and `LDFLAGS`, respectively.

Reply:
Fixed in rebased commit.